### PR TITLE
Sync next

### PR DESCRIPTION
**Pull request by Baroshem**
### Related Issues

closes #

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- I tested manually my code and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme
- [ ] I have written test cases for my code

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

File: packages/core/nuxt-module/lib/module.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const rawSourcesModule = require('./modules/raw-sources-loader');
 module.exports = function VueStorefrontNuxtModule (moduleOptions) {
```
Comment (Mateuszpietrusinski) [inline packages/core/nuxt-module/lib/module.js:11]: Just asking why `lib` folder is included in the repo?

File: packages/core/nuxt-module/lib/plugins/context.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
-
-import { configureContext } from '@vue-storefront/core'
+import { configureContext } from '@vue-storefront/core';
```
Comment (Mateuszpietrusinski) [inline packages/core/nuxt-module/lib/plugins/context.js:1]: same as above

**Review by Mateuszpietrusinski — CHANGES_REQUESTED** [review]
Most of the files LGTM, but I saw that this pr includes changes inside `lib` folder in `nuxt-module` package, let me know is it know behavior?

**Comment by Mateuszpietrusinski** [comment]
Ok after analyze I realize that those fields are added to the repository in Feb 2020 so let's leave them in peace.

**Review by Mateuszpietrusinski — APPROVED** [review]

**Review by Fifciu — APPROVED** [review]

**Comment by Fifciu** [comment]
LGTM

**Review by igorwojciechowski — APPROVED** [review]

**Pull request merged**